### PR TITLE
feat: add "clamp to ground" option to file primitive

### PR DESCRIPTION
**Pull request by HideBa**
# Overview

### CODE REVIEW

File: pkg/builtin/manifest.yml [hunk 1]
```diff
@@ -1144,6 +1144,11 @@ extensions:
                   label: GeoJSON / TopoJSON
                 - key: czml
                   label: CZML
+            - id: clampToGround
+              type: bool
+              title: Clamp to ground
+              description: Option to clamp to ground or not.
```
Comment (KaWaite) [inline pkg/builtin/manifest.yml:1150]:
```suggestion
              description: Option to attach the primitive to the ground (keeps primitive on top of the map even with terrain enabled).
```

Comment (KaWaite) [reply pkg/builtin/manifest.yml:1150]: Please check that the description is accurate!

**Review by KaWaite — APPROVED** [review]
Updated English. Check accuracy of my updated description before committing

File: pkg/builtin/manifest_ja.yml [hunk 2]
```diff
@@ -510,6 +510,9 @@ extensions:
               kml: KML
               geojson: GeoJSON / TopoJSON
               czml: CZML
+          clampToGround:
+            title: 地表に固定
+            description: 外部データを地表に合わせて固定します。
```
Comment (rot1024) [inline pkg/builtin/manifest_ja.yml:515]:
```suggestion
            description: 地表の凹凸に沿うようにポリゴンなどのデータを表示します。
```

**Review by rot1024 — CHANGES_REQUESTED** [review]

**Pull request merged**